apps/fl/js/utils.js

```diff
@@ -15,6 +25,16 @@ function _(id, args) {
 
 // Show a dialog with one or two buttons
 function showDialog(options) {
+
+  function close(e) {
+    document.body.removeChild(dialog);
+    e.preventDefault();
+    e.stopPropagation();
+  }
```

Comment:
Nope, it really complains =(

The other option is to declare a var at the top, and assign it later on. E.g.,

```
var close;
function myFunc() {
  // Code that references close.
}
close = function() {};
myFunc();
```

I don't think that is much better, so I think we should go with this for now. Assigning functions to some containing object is also another workaround, but this is the smaller change,